template-theory/css/medium-screen.css

```
@@ -0,0 +1,73 @@
+@media screen and (max-width: 1280px) {
+    .content{
+        margin-top: 0px;
+        width: 90%;
+    }
+
+    div.actitcle {
+        border: none;
+        margin-left: 0px;
+    }
+    .actitcle .title-actitcle{
+        margin-top: 1.5em;
+    }
+    .actitcle .actitcle-footer {
+        padding-bottom: 2em;
+        border-bottom: 1px solid #dbdbdb;
+    }
+    .actitcle .title-actitcle,
+    .actitcle .actitcle-discripton,
+    .actitcle .actitcle-footer {
+        width: 100%;
+    }
+    .col-lg-4:last-of-type .actitcle .actitcle-footer{
```

```suggestion
    .col-lg-4:last-of-type .actitcle .actitcle-footer {
```
Không query theo `.col-lg-4`, `.col-lg-4` chỉ để đi chia grid, muốn css cho `.actitcle-footer` thì hãy đặt thêm 1 class khác vào